contracts/staking/ServiceStakingBase.sol

```diff
@@ -519,16 +567,20 @@ abstract contract ServiceStakingBase is ERC721TokenReceiver, IErrorsRegistries {
         // Get the unstaked service data
         uint256 reward = sInfo.reward;
         uint256[] memory nonces = sInfo.nonces;
-        uint256 tsStart = sInfo.tsStart;
         address multisig = sInfo.multisig;
 
+        // TODO: Add inactivity to the global map of inactivities, in order to never let stake if that limit has been breached
```

Comment:
We didn't discuss this. Right now evicted services will be slashed so there's no reason to perma-ban them.

Reply:
(which is not sybil resistant anyway)

Reply:
It was left from the previous implementation.